peps/pep-0776.rst

```diff
@@ -0,0 +1,751 @@
+PEP: 776
+Title: Emscripten Support
+Author: Hood Chatham <roberthoodchatham at gmail.com>
+Sponsor: Łukasz Langa <lukasz at python.org>, Russell Keith-Magee <russell at keith-magee.com>
```

Comment:
```
peps/pep-0776.rst:4:  Sponsor entries must not contain multiple '<': 'Łukasz Langa <lukasz at python.org>,\n         Russell Keith-Magee <russell at keith-magee.com>'
```

Hmm, I think the linter is only expecting a single sponsor.

https://peps.python.org/pep-0001/#submitting-a-pep talks of a singular sponsor.